Kitodo-API/src/main/java/org/kitodo/config/OPACConfig.java

```diff
@@ -135,4 +139,8 @@ private static XMLConfiguration getConfig() {
         config.setReloadingStrategy(new FileChangedReloadingStrategy());
         return config;
     }
+
+    public static String getParentElementXPath(String catalogName) {
```

Comment:
Public method without JavaDoc

Reply:
Is the missing JavaDoc a bigger problem than the method not being used at all? ;)